src/Umbraco.Tests/Testing/TestOptionAttributeBase.cs

```diff
@@ -8,6 +9,8 @@ namespace Umbraco.Tests.Testing
 {
     public abstract class TestOptionAttributeBase : Attribute
     {
+        public static readonly List<string> ScanAssemblies = new List<string>();
```

Comment:
Does this need to be a string? Why not just have it as a list of `Assembly` which would make it simpler to use?